builtin/providers/aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -152,6 +191,75 @@ resource "aws_elasticache_cluster" "bar" {
 }
 `, genRandInt(), genRandInt(), genRandInt())
 
+var testAccAWSElasticacheClusterConfig_snapshots = `
+provider "aws" {
+	region = "us-east-1"
+}
+resource "aws_security_group" "bar" {
+    name = "tf-test-security-group-%03d"
+    description = "tf-test-security-group-descr"
+    ingress {
+        from_port = -1
+        to_port = -1
+        protocol = "icmp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+}
+
+resource "aws_elasticache_security_group" "bar" {
+    name = "tf-test-security-group-%03d"
+    description = "tf-test-security-group-descr"
+    security_group_names = ["${aws_security_group.bar.name}"]
+}
+
+resource "aws_elasticache_cluster" "bar" {
+    cluster_id = "tf-test-%03d"
+    engine = "redis"
+    node_type = "cache.t2.small"
```

Comment:
this needs to be changed to `cache.m1.small`, or `snapshot_window` and `snapshot_retention_limit` need to be removed. The test that uses this config doesn't pass